### PR TITLE
Fix to prevent panic on looping through empty vector

### PR DESCRIPTION
**Pull request by dogeatdog**
I didn't have access to create a PR through my desktop, so describing it like this. 
In browser/tab/element/mod.rs I've added a check around the loop (line 80) : 

`
```
if attrs_vec.len() > 0 {
            for i in (0..attrs_vec.len() - 1).step_by(2) {
                let k = attrs_vec[i].to_string();
                let v = attrs_vec[i+1].to_string();
                attrs.insert(k, v);
            }
        }
````

That will prevent a panic when there are no elements in the vector. (overflow in dev, index out of bounds in release.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+version = "1.0.2"
 authors = ["Alistair Roche <alistair@sunburnt.country>"]
 edition = "2018"
 description = "Control Chrome programatically"
 license = "MIT"
-homepage = "https://github.com/atroche/rust-headless-chrome"
-repository = "https://github.com/atroche/rust-headless-chrome"
+homepage = "https://github.com/masc-it/rust-headless-chrome"
+repository = "https://github.com/masc-it/rust-headless-chrome"
 readme = "README.md"
```
Comment (mdrokz) [inline Cargo.toml:10]: can you please revert this ?

File: Cargo.toml [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
 [dependencies]
-websocket = { version = "0.23", default_features = false, features = ["sync"] }
+websocket = { version = "0.24", default_features = false, features = ["sync"] }
```
Comment (mdrokz) [inline Cargo.toml:13]: the latest master uses tungstenite can you use deps from master ?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Headless Chrome
 
 [![Build Status](https://travis-ci.com/atroche/rust-headless-chrome.svg?branch=master)](https://travis-ci.com/atroche/rust-headless-chrome)
-[![Crate](https://img.shields.io/crates/v/headless_chrome.svg)](https://crates.io/crates/headless_chrome)
+[![Crate](https://img.shields.io/crates/v/headless_chrome.svg)](https://crates.io/crates/headless_chrome_fork)
```
Comment (mdrokz) [inline README.md:4]: revert this as well

**Review by mdrokz — CHANGES_REQUESTED** [review]
so far it looks good can you explain the storing document change ? and please revert the changes to `Cargo.toml` and `README.md`

**Comment by mdrokz** [comment]
there are some conflicts as well can you resolve them properly ?

**Comment by dogeatdog** [comment]
I goofed up. I was actually trying to create a PR to the forked repo https://github.com/masc-it/rust-headless-chrome, but somehow totally got lost in github. Hence the reason for the weird PR and the fact I just closed it right now. 

Sorry for the confusion !

**Pull request closed without merging**